### PR TITLE
commit message

### PR DESCRIPTION
**Pull request by alex-min**

### CODE REVIEW

**Comment by chrisblackburn** [comment]
Interesting

File: README.md [hunk 1]
```diff
@@ -22,3 +22,4 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
```
Comment (chrisblackburn) [inline README.md:24]: Rubbish

**Comment by alex-min** [comment]
[BS]

**Comment by alex-min** [comment]
:fire:

**Comment by emileswarts** [comment]
walalalalalalala nice code [BS]

**Pull request closed without merging**